lib/spec/runners/operations.ex

```diff
@@ -91,6 +91,15 @@ defmodule OperationsTestRunner do
     assert true
   end
 
+  defp handle_case("attester_slashing", pre, attester_slashing, post, _case_dir) do
```

Comment:
This shouldn't be needed anymore once it's synced with main

Reply:
I will be removing it just before the merge since I keep testing while I am changing the codebase

Reply:
Disregard the last comment. I thought this was the `epoch_processing` runner 😅